select-figure/main.ipynb

```diff
@@ -55,44 +55,41 @@
     "    import pandas\n",
     "    pandas.set_option('max_colwidth', 1000)\n",
     "\n",
-    "    # label, ref_uuid, opt_uuid \n",
-    "    uuids = get_data_aiida()\n",
-    "    uuid_dict = { v[0]: v[1:] for v in uuids }\n",
-    "\n",
-    "    # label, ref_uuid, opt_uuid, pe_uuid\n",
-    "    uuids_pe = get_data_aiida(has_pe=True)\n",
-    "    uuid_dict_pe = { v[0]: v[1:] for v in uuids_pe }\n",
-    "\n",
+    "    # import cof-frameworks as DataFrame\n",
+    "    url = 'https://raw.githubusercontent.com/danieleongari/CURATED-COFs/additions/cof-frameworks.csv' #TODO: update\n",
+    "    df = pandas.read_csv(url) # DEPRECATED: pandas.read_csv(this_dir + \"static/cof-frameworks.csv\")\n",
```

Comment:
this is something I wouldn't recommend.
not only can this slow down things, it also means existing apps can break whenever we decide to change something about the format here.

once we move to a pipeline for updating things on the fly, we can revisit these things, but for the moment let's keep a local copy